lxd/daemon.go

```diff
@@ -1602,6 +1603,17 @@ func (d *Daemon) init() error {
 		d.globalConfig = globalConfig
 		d.globalConfigMu.Unlock()
 
+		// Add the per-project config options to the daemon config schema.
+		projects, err := dbCluster.GetProjectNames(ctx, tx.Tx())
+		if err != nil {
+			return err
```

Comment:
please can you wrap errors, e.g. `return fmt.Error("Failed getting projects: %w", err)` so that if it does fail we get a more useful error to debug client installs.